### PR TITLE
Issue #466: Add upstream remote in setup script

### PR DESCRIPTION
**Pull request by pascalchevrel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,14 +57,23 @@ Note that if you launch start.sh again after the installation, it will not downl
 
 ## Contribution tips
 
-- Add Transvision parent project as remote:
-"git remote add transvision git@github.com:mozfr/transvision.git"
+- Add Transvision parent project as a remote repository called 'upstream':
```
Comment (flodolo) [inline README.md]: I would add that this is needed only once at the beginning.

File: README.md [hunk 2]
```diff
@@ -57,14 +57,23 @@ Note that if you launch start.sh again after the installation, it will not downl
...
+```bash
+git remote add upstream git@github.com:mozfr/transvision.git
+```
 - Update your branch to the last version of Transvision:
```
Comment (flodolo) [inline README.md]: Same here, but explaining that it has to be done at least every time you start doing some development work.

File: README.md [hunk 3]
```diff
@@ -57,14 +57,23 @@ Note that if you launch start.sh again after the installation, it will not downl
...
-"git pull transvision master"
+```bash
+git pull upstream master
```
Comment (flodolo) [inline README.md]: Interesting, I use 
```
git fetch upstream
git merge upstream/master
```

I'add a `git checkout master` before that to make sure the repo is on the right branch.

File: app/scripts/dev-setup.sh [hunk 4]
```diff
@@ -127,3 +127,20 @@ then
...
+    origin=$(git config --get remote.origin.url)
+    remote_https='https://github.com/mozfr/transvision.git'
+    remote_git='git@github.com:mozfr/transvision.git'
+
+    if [ $origin == $remote_https ] || [ $origin == $remote_git ]
+    then
+        echored "Your local clone is from the reference repository, you should clone your own fork if you intend to contribute code."
+    else
+        echogreen "${remote_git} added as upstream remote"
+        git remote add upstream ${remote_git}
```
Comment (flodolo) [inline app/scripts/dev-setup.sh]: PHP syntax? In both lines it should be ```$var_name```

File: README.md (hunk 3 above)
Comment (pascalchevrel) [inline README.md]: http://git-scm.com/docs/git-pull
git pull is a shorthand for git fetch+merge

Comment (flodolo) [inline README.md]: Yep, not sure why I never switch to ```git pull```, force of habit.

**Pull request merged**